release-1.8/release_notes_draft.md

```diff
@@ -165,68 +165,43 @@ across the system. Here's the release [scalability validation report].
+  * The audit log file defaults to JSON encoding when using the advanced auditing feature gate.
+  * The `--audit-policy-file` option requires `kind` and `apiVersion` fields specifying what format version the `Policy` is using.
   * The webhook and log file now output the `v1beta1` event format.
-  * For more details, see [Advanced audit](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#advanced-audit)
 
-* The deprecated ThirdPartyResource (TPR) API has been removed.
-  To avoid losing your TPR data, you must
-  [migrate to CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/migrate-third-party-resource/)
-  **before upgrading to 1.8**.
+    For more details, see [Advanced audit](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#advanced-audit).
+
+* The deprecated `ThirdPartyResource` (TPR) API was removed.
+  To avoid losing your TPR data, [migrate to CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/migrate-third-party-resource/).
 
-* The following deprecated flags have been removed from `kube-controller-manager`:
+* The following deprecated flags were removed from `kube-controller-manager`:
 
   * `replication-controller-lookup-cache-size`
   * `replicaset-lookup-cache-size`
   * `daemonset-lookup-cache-size`
 
-  Do not use deprecated flags.
-
-* StatefulSet: The deprecated `pod.alpha.kubernetes.io/initialized` annotation for
-  interrupting StatefulSet Pod management is now ignored. If the annotation is set
-  to `true` or left blank, you won't see any change in behavior. If it's set to
-  `false`, previously dormant StatefulSets might become active after upgrading.
-
-* CronJobs has been promoted to `v1beta1` which is now turned on by default.
-  Although version `v2alpah1` is still available, it is deprecated.  Migrate to
-  `batch/v1beta1.CronJobs`.  Additionally, upgrading cluster in high-availability
-  configuration might return errors. The new controllers rely on the latest
-  version of the resources.  If the expected version is not found during rolling
-  upgrade, the system throws resource not found errors.
-* The `batch/v2alpha1.ScheduledJobs` has been removed.  Migrate to `batch/v1beta.CronJobs`
-  to continue managing time based jobs.
-
-* The APIs `rbac/v1alpha1`, `settings/v1alpha1`, and `scheduling/v1alpha1` are
-  disabled by default.
-
-* The `system:node` role is no longer automatically granted to the `system:nodes`
-  group in new clusters. It is recommended that nodes be authorized using the
-  `Node` authorization mode instead. Installations that wish to continue giving
-  all members of the `system:nodes` group the `system:node` role (which grants
-  broad read access, including all secrets and configmaps) must create an
-  installation-specific `ClusterRoleBinding`. ([#49638](https://github.com/kubernetes/kubernetes/pull/49638))
+  It is recommended that you don't use them.
```

Comment:
What happens if you DO use them?  Are they ignored or will they cause errors?

Reply:
Got the info. If they use a flag that is deprecated, but not yet removed, the server will print/log a warning. If they use a flag that is removed, the server will abort startup. Let me add that to the rn.